### PR TITLE
docs: fix minor typos, phrasing, outdated props

### PR DESCRIPTION
**Pull request by andretchen0**
I've updated some of the docs here. Mostly correcting minor typos and adjusting some phrasing. 

The PR also updates [outdated props names/defaults in the table here.](https://cientos.tresjs.org/guide/abstractions/mouse-parallax.html#props)

Let me know if this sort of fix is welcome. If so, I'll keep going. If not, then I'll stop. No problem either way.

----

I'm not an editor. My only relevant qualification is being an anglophone. 🤪

### CODE REVIEW

File: docs/guide/misc/stats.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Stats
 
-[stats.js](https://github.com/mrdoob/stats.js/) is a JavaScript performance monitor, made by [mrdoop](https://github.com/mrdoob). stats.js provides a simple info box that will help you monitor your code performance.
+[stats.js](https://github.com/mrdoob/stats.js/) is a JavaScript performance monitor, made by [mrdoob](https://github.com/mrdoob). stats.js provides a simple info box that will help you monitor your code performance.
```
Comment (JaimeTorrealba) [inline docs/guide/misc/stats.md:3]: This mistake ... hahaha

File: docs/guide/abstractions/mouse-parallax.md [hunk 2]
```diff
@@ -4,13 +4,13 @@
   <MouseParallaxDemo />
 </DocsDemo>
 
-`<MouseParallax />` is a component that allow you to create easily the pam parallax effect. The camera will update automatically according to the mouse position, creating a beautiful nice effect
+`<MouseParallax />` is a component that allows you to easily create a [parallax](https://en.wikipedia.org/wiki/Parallax) effect. The camera will update automatically according to the mouse position.
 
 ## Usage
 
-You only need import it and use it `<MouseParallax />` additionally you can pass two props, ease and factor.
+You only need import it and add it to your template as `<MouseParallax />`. Additionally, you can pass the following props.
```
Comment (JaimeTorrealba) [inline docs/guide/abstractions/mouse-parallax.md:11]: to import

the following props:

**Review by JaimeTorrealba — CHANGES_REQUESTED** [review]
Thank you for this, I re-read some texts and find some little improvements

**Comment by JaimeTorrealba** [comment]
Of course, these types of fix are welcome and appreciated :)

Are you in discord? Or is there any more direct way to contact to you @andretchen0?

**Comment by andretchen0** [comment]
> Are you in discord? 

I'm not on Discord very often; I just joined the server. I have to be a member for at least 10 minutes to send a message. ;)

File: docs/guide/misc/stats.md (hunk 1 above)
Comment (andretchen0) [reply docs/guide/misc/stats.md:3]: Lol. Yeah. This whole PR was really just to fix that.

**Comment by andretchen0** [comment]
Hey @JaimeTorrealba I made the wording changes you pointed out above.

Following the quick exchange on Discord, I added a "Usage" header where it was missing. In a few places, that meant reordering things a bit.

Since I was looking at usage examples, I also reformatted a lot of them – mostly cosmetic stuff. Especially:

* changing the "language" of the example – e.g., `html` -> `vue`
* changing the highlighted lines so that the relevant lines are emphasized

File: docs/guide/abstractions/mouse-parallax.md (hunk 2 above)
Comment (andretchen0) [reply docs/guide/abstractions/mouse-parallax.md]: Done

File: docs/guide/controls/keyboard-controls.md [hunk 3]
```diff
@@ -2,18 +2,19 @@
...
 
-## In combination with PointerLockControls.
+## Usage
+
+### In combination with PointerLockControls
 
 Here you need to use the PointerLockControls with make-default props, KeyboardControls will automatically adapt to it, creating a really good effect first player experiences.
 
-```vue{3}
+```vue{5}
```
Comment (JaimeTorrealba) [inline docs/guide/controls/keyboard-controls.md:11]: Why Vue 5 here?? (I'm not good with markdown)

Comment (andretchen0) [reply docs/guide/controls/keyboard-controls.md:11]: > Why Vue 5 here?? (I'm not good with markdown)

No problem. It took me some searching to find the docs for the extensions.

That extension is for [syntax and line highlighting.](https://vitepress.dev/guide/markdown#line-highlighting-in-code-blocks) 

Using the extension, this:

```
```vue{5}
```

... means:

* Begin codeblock. 
* Use Vue syntax highlighting. 
* Highlight line 5.

**Comment by andretchen0** [comment]
@JaimeTorrealba 

Merging is blocked for me, but I can't find any more requested changes. Can you review and merge if everything is ok? Thanks!

**Review by JaimeTorrealba — APPROVED** [review]

**Pull request merged**